admin/index.php

```diff
@@ -90,7 +90,14 @@ function bounceOut(){ header("Location: http://davesmapper.com/admin"); exit; }
 	}
 	if ($_POST['action'] == "add_carto") {
 		mysql_query("INSERT INTO artists (name, url_slug, initials, icon, url, bio, email, password)
-								 VALUES ('".mysql_real_escape_string($_POST['aname'])."', '".mysql_real_escape_string($_POST['aurlslug'])."', '".mysql_real_escape_string($_POST['ainit'])."', '".mysql_real_escape_string($_POST['aicon'])."', '".mysql_real_escape_string($_POST['alink'])."', '".mysql_real_escape_string($_POST['abio'])."', '".mysql_real_escape_string($_POST['aemail'])."', '".sha1($_POST['aemail']."bacon"."c00lt1l3sbr0")."')");
+								 VALUES ('".mysql_real_escape_string($_POST['aname'])."',
+								 				 '".mysql_real_escape_string($_POST['aurlslug'])."',
+								 				 '".mysql_real_escape_string($_POST['ainit'])."',
+								 				 '".mysql_real_escape_string($_POST['aicon'])."',
+								 				 '".mysql_real_escape_string($_POST['alink'])."',
+								 				 '".mysql_real_escape_string($_POST['abio'])."',
+								 				 '".mysql_real_escape_string($_POST['aemail'])."',
+								 				 '".sha1($_POST['aemail']."bacon".$_POST['apass'])."')");
```

Comment:
I think you can reduce the horizontal spacing on these a little bit to keep them from scrolling off the screen when viewing the diff on github. Don't worry, there's a lot of bad ways to do it, a few mediocre ways, and no good ways :)